api/room/schema.py

```diff
@@ -147,7 +147,32 @@ def mutate(self, info, room_id, **kwargs):
         return DeleteRoom(room=exact_room)
 
 
+class IncrementTotalRoomFeedbackView(graphene.Mutation):
+
+    class Arguments:
+        room_id = graphene.Int(required=True)
+    room = graphene.Field(Room)
+
+    def mutate(self, info, **kwargs):
+        validate_empty_fields(**kwargs)
+        query = Room.get_query(info)
+        exact_room = query.filter_by(id=kwargs['room_id']).first()
+        if not exact_room:
+            raise GraphQLError("Non-existent room id")
+
+        if exact_room.total_room_feedback_views:
+            new_total_feedback_view = exact_room.total_room_feedback_views + 1
+        if not exact_room.total_room_feedback_views:
+            new_total_feedback_view = 1
+        update_entity_fields(
+            exact_room, total_room_feedback_views=new_total_feedback_view
+        )
+        exact_room.save()
+        return IncrementTotalRoomFeedbackView(room=exact_room)
+
+
 class Mutation(graphene.ObjectType):
     create_room = CreateRoom.Field()
     update_room = UpdateRoom.Field()
     delete_room = DeleteRoom.Field()
+    increment_room_feedback = IncrementTotalRoomFeedbackView.Field()
```

Comment:
@RonKbS any reason why you did not just use the update_room mutation?

Reply:
I have looked through the two pieces of code and realized there is indeed a similarity that would point to me having to include these changes in the update_room mutation.

However, the idea of incrementing responses (for a room's feedback question) is separate from updating a room I think, so I hesitate to do this. I will defer to your decision on the matter.

Reply:
@RonKBS except for obvious reasons, I think this should go to the update_room_mutation. You're actually editing a resource in the room, it doestn't matter if it is a different story/task.

Reply:
Definitely, this should be part of updating room details, I also think that the admin should be able to see the total number of feedback views when they query a particular room details.

Reply:
Alright,

I have begun working on an implementation that puts these changes in the `update_room` mutation.